### PR TITLE
Adding new repository for indexing: jason-ros

### PR DESCRIPTION
**Pull request by Rezenders**

### CODE REVIEW

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -3811,6 +3811,11 @@ repositories:
       url: https://github.com/jackal/jackal_simulator.git
       version: kinetic-devel
     status: maintained
+  jason-ros:
```
Comment (clalancette) [inline melodic/distribution.yaml]: We generally prefer underscores, so this should be: `jason_ros`.

File: melodic/distribution.yaml [hunk 2]
```diff
@@ -3811,6 +3811,11 @@ repositories:
...
+    doc:
+      type: git
+      url: https://github.com/jason-lang/jason-ros
```
Comment (clalancette) [inline melodic/distribution.yaml]: This is failing CI because it needs to end in `.git`.

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by nuclearsandwich** [comment]
@Rezenders do you intend to follow up on the requested changes or can we close this PR for now?

**Comment by nuclearsandwich** [comment]
Closing the PR for now but feel free to re-open when you're ready to update it @Rezenders

**Comment by Rezenders** [comment]
Sorry about the delay. I fixed the issues and opened a new PR #23999

**Pull request closed without merging**